lab-khalid/test/test_employee_routes.js

```diff
@@ -0,0 +1,74 @@
+'use strict';
+
+const expect = require('chai').expect;
+const request = require('superagent');
+require('../server.js');
+
+describe('Employee routes', function(){
+  var employee = null;
+  describe('GET /api/fakeroute', function(){
+    it('should return a 404', function(done){
+      request.get('localhost:8000/api/fakeroute/374734736473')
+      .end((err, res) => {
+        expect(!!err).to.equal(true);
```

Comment:
Is there a reason you specifically are wanting to test whether an error is equal to true? This seems like a bit of an unnecessary test and it would be a better test to run something like `err.message`.to.equal('Not Found')`.